### PR TITLE
fix(docs) Updated the name from v2 to OIP

### PR DESCRIPTION
**Pull request by Rajakavitha1**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:
I have not updated the references in https://github.com/SeldonIO/seldon-core/blob/f993cbb2c96043168ee59b1d6386ec3880f42eb3/notebooks/explainer_examples_v2.ipynb. This I hope the developer does.

### CODE REVIEW

File: doc/source/analytics/explainers.md [hunk 1]
```diff
@@ -36,9 +36,9 @@ For Alibi explainers that need to be trained you should
 
 The runtime environment in our [Alibi Explain Server](https://github.com/SeldonIO/seldon-core/tree/master/components/alibi-explain-server) is locked using [Poetry](https://python-poetry.org/). See our e2e example [here](../examples/iris_explainer_poetry.html) on how to use that definition to train your explainers.
 
-### V2 protocol for explainer using [MLServer](https://github.com/SeldonIO/MLServer) (incubating)
+### Open Inference Protocol for explainer using [MLServer](https://github.com/SeldonIO/MLServer) (incubating)
```
Comment (matthewlowdon) [inline doc/source/analytics/explainers.md:39]: Is it incubating?

**Review by matthewlowdon — APPROVED** [review]
Fine, but I added one comment to consider.

File: doc/source/analytics/explainers.md (hunk 1 above)
Comment (Rajakavitha1) [reply doc/source/analytics/explainers.md:39]: @matthewlowdon  Please suggest what it should be. I just changed only the name in that line.

Comment (lc525) [inline doc/source/analytics/explainers.md:39]:
```suggestion
### Open Inference Protocol for explainer using [MLServer](https://github.com/SeldonIO/MLServer)
```

File: doc/source/examples/notebooks.rst [hunk 2]
```diff
@@ -22,7 +22,7 @@ Prepackaged Inference Server Examples
    Deploy a Scikit-learn Model Binary <../servers/sklearn.md>
    Deploy a Tensorflow Exported Model <../servers/tensorflow.md>
    MLflow Pre-packaged Model Server A/B Test <mlflow_server_ab_test_ambassador>
-   MLflow v2 Protocol End to End Workflow (Incubating) <mlflow_v2_protocol_end_to_end>
+   MLflow Open Inference Protocol End to End Workflow (Incubating) <mlflow_v2_protocol_end_to_end>
```
Comment (lc525) [inline doc/source/examples/notebooks.rst:25]:
```suggestion
   MLflow Open Inference Protocol End to End Workflow <mlflow_v2_protocol_end_to_end>
```

File: doc/source/examples/notebooks.rst [hunk 3]
```diff
@@ -90,7 +90,7 @@ Advanced Machine Learning Monitoring
 
    Real Time Monitoring of Statistical Metrics <feedback_reward_custom_metrics>
    Model Explainer Example <iris_explainer_poetry>
-   Model Explainer V2 protocol Example (Incubating) <iris_anchor_tabular_explainer_v2>
+   Model Explainer Open Inference Protocol Example (Incubating) <iris_anchor_tabular_explainer_v2>
```
Comment (lc525) [inline doc/source/examples/notebooks.rst:93]:
```suggestion
   Model Explainer Open Inference Protocol Example <iris_anchor_tabular_explainer_v2>
```

**Review by lc525 — APPROVED** [review]
Added suggestions to remove "incubating" in a number of places. The OIP is now established and the only thing we'd like to support moving forward.

File: doc/source/analytics/explainers.md (hunk 1 above)
Comment (Rajakavitha1) [reply doc/source/analytics/explainers.md]: Fixed now. Incorporated the suggestions that @lc525  made.

**Pull request merged**